.travis.yml

```diff
@@ -38,14 +31,36 @@ before_install:
   - npm config set cache $HOME/.npm-cache --global
   - npm config set prefix $HOME/.npm-prefix --global
 
-install:
-  - time mvn install -T 2C -q -DskipTests=true -Dmaven.javadoc.skip=true -B -V
-
-script:
-  - time $SCRIPT
-
 matrix:
   fast_finish: true
+  include:
+    - name: Unit Tests
+      script:
+        - time mvn install -T 2C -q -DskipTests=true -Dmaven.javadoc.skip=true -B -V -pl '!:metron-config,!:metron-alerts'
+        - time mvn surefire:test@unit-tests -T 2C
+
+    - name: Integration Tests
+      script:
+        - time mvn install -T 2C -q -DskipTests=true -Dmaven.javadoc.skip=true -B -V -pl '!:metron-config,!:metron-alerts'
+        - time mvn surefire:test@integration-tests
+
+    - name: Alerts UI Tests
+      script:
+        - time mvn install -T 2C -q -DskipTests=true -Dmaven.javadoc.skip=true -B -V -pl ':metron-alerts'
```

Comment:
I don't think I realized you could use the colons for the `-pl` list as well (I always use it with `-rf`). I like that. Way cleaner than typing the full module path.